README.md

```diff
@@ -83,3 +83,87 @@ A script to do some basic QC is provided in `bin/qc.py`. It measures the % of re
 
 #### Output
 A subdirectory for each process in the workflow is created in `--outdir`. A `nml_upload` subdirectory containing dehosted fastq files and consensus sequences is included. 
+
+#### Provenance
+In the output directory for each sample, a provenance file will be written with the following format:
+```
+- process_name: performHostFilter
+  tools:
+    - tool_name: bwa mem
```

Comment:
Sorry this is a nit-pick but this should be:

```
- tool_name: bwa
```

Reply:
fixed!